x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/rules/50_query_rule_put.yml

```diff
@@ -0,0 +1,114 @@
+setup:
```

Comment:
Note: There's not a lot of validation here. These will be fleshed out more in a followup PR when the `GET` and `DELETE` endpoints are implemented.